apps/dg/components/case_card/case_card.js

```diff
@@ -509,7 +551,7 @@ DG.React.ready(function () {
                     isEditing: tAttrID === this.state.attrIdOfValueToEdit,
                     onToggleEditing: toggleEditing,
                     onEscapeEditing: escapeEditing,
-                    editModeCallback: editModeCallback
+                    onEditModeCallback: editModeCallback
```

Comment:
This typo should mean there's a bug in editing case values, unless this doesn't matter for some reason. 🤔